server/pom.xml

```diff
@@ -186,6 +186,10 @@
             <groupId>org.bitbucket.b_c</groupId>
             <artifactId>jose4j</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Jeg liker Brian Campbell sin versjon bedre (bitbucket.b_c) - den er skrapet ned og gjør det du trenger (Jose) med svakt forskjellig syntaks. BC bidrar forsåvidt til nimbus så den er trygg.

Reply:
Se fx min PR for hvordan generere signert token med BC

Reply:
VTP bruker såpass få imports (3) av nimbus at det burde holde med org.jose4j (BC)